sdk/resourcemanager/azure-resourcemanager-containerservice/README.md

```diff
@@ -18,7 +18,7 @@ For documentation on how to use this package, please see [Azure Management Libra
 <dependency>
     <groupId>com.azure.resourcemanager</groupId>
     <artifactId>azure-resourcemanager-containerservice</artifactId>
-    <version>2.27.0</version>
+    <version>2.28.0-beta.1</version>
```

Comment:
Don't change this. It will be done upon release.

Reply:
The code has been rollbacked in the new version.